### PR TITLE
Dependency upgrades for upcoming 7.x backport

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test this

**Comment by ljacomet** [comment]
@bot-gradle test this

File: gradle/verification-keyring.keys [hunk 1]
```diff
@@ -14946,3 +14946,202 @@ JNQ80WoyN7AJ7ZqUyi/SzQfS6T+DXVohQPEzzBByIEqjLr9Sc8nJ4I3dGa/Y6Eib
...
+c8peel2SlqFLQVZ+3Lb8JKC/9DeHAGmpBipB+VcSC5vRchqpncsHAz0GXtmqzkgc
+yzq0Kyh0nIshDsGQ/vXoZQRl5hjMWBLaN/y5ik7Qde4bjL7p1ozKVm0S2qY0CfrG
+SzZnLXLOxX90fqXXN9cwmLv++bdyrXElA10u9LIqsViop3UvHitVjkJlIhwt9/du
+JEhPJnT/qH2O9YAqwlEFggSNSctD3LC1uMjB8JD6XcpQgDP6BiZUwP/yCGQGvA27
+Omf2WgrpGukXiZFSBkCZPL+/rcP5dnVrCwneGCXluI+sONfybagy2nNTTJWxonBh
+7c5p
+=sqwf
+-----END PGP PUBLIC KEY BLOCK-----
+
+pub    54AC8E2D98CFEAC6
```
Comment (ov7a) [inline gradle/verification-keyring.keys:14997]: NIT: Consider removing identical keys

File: gradle/verification-keyring.keys [hunk 2]
```diff
@@ -14946,3 +14946,202 @@ JNQ80WoyN7AJ7ZqUyi/SzQfS6T+DXVohQPEzzBByIEqjLr9Sc8nJ4I3dGa/Y6Eib
...
```
Comment (ov7a) [inline gradle/verification-keyring.keys:15044]: Another duplicate

File: gradle/verification-metadata.xml [hunk 3]
```diff
@@ -1437,15 +1439,11 @@
...
-      <component group="org.yaml" name="snakeyaml" version="1.32">
-         <artifact name="snakeyaml-1.32.jar">
-            <pgp value="120d6f34e627ed3a772ebbfe55c7e5e701832382"/>
-         </artifact>
-      </component>
       <component group="org.ysb33r.gradle" name="grolifant" version="0.16.1">
          <artifact name="grolifant-0.16.1.jar">
             <ignored-keys>
                <ignored-key id="54ac8e2d98cfeac6" reason="also-trust requires ignoring signatures"/>
+               <ignored-key id="ea022560a81e5bd48db3d18b54ac8e2d98cfeac6" reason="PGP verification failed"/>
```
Comment (ov7a) [inline gradle/verification-metadata.xml:1446]: ❓ Why is it ignored?

Comment (ljacomet) [reply gradle/verification-metadata.xml:1446]: That looks like a bug in our generation.
The key is already ignored, with its justification just above. Only it is done with the long id and not the full fingerprint.
I'll remove the addition and replace the long id with a full fingerprint.

File: gradle/verification-keyring.keys (hunk 1 above)
Comment (ljacomet) [reply gradle/verification-keyring.keys:14997]: Removed

**Review by ov7a — APPROVED** [review]

**Pull request merged**